src/components/Registration/RegAddEditOrgPerson.vue

```diff
@@ -250,7 +250,11 @@
                 If you have an existing business number, enter it below and we will contact
                 Canada Revenue Agency and ask them to link it to this registration.
               </p>
-              <HelpBusinessNumber class="mt-4" />
+              <HelpBusinessNumber
+                class="mt-4"
+                :isTypeSoleProp="isTypeSoleProp"
+                :isTypePartnership="isTypePartnership"
```

Comment:
Passing in some props so HelpBusinessNumber doesn't have to know about the Store.